packages/hash/frontend/src/components/PageIcon.tsx

```diff
@@ -55,53 +38,25 @@ export const PageIcon = ({
   const sizes = variantSizes[size];
 
   return (
-    <>
-      <Tooltip title="Change icon" placement="bottom">
-        <IconButton
-          {...bindTrigger(popupState)}
-          sx={({ palette }) => {
-            const background = hasDarkBg ? palette.gray[40] : palette.gray[30];
-
-            return {
-              width: sizes.container,
-              height: sizes.container,
-              fontSize: sizes.font,
-              ...(popupState.isOpen && {
-                background,
-              }),
-              "&:focus-visible, &:hover": {
-                background,
-              },
-              [`&.${iconButtonClasses.disabled}`]: { color: "unset" },
-            };
-          }}
-          disabled={readonly || updateEntityLoading}
-        >
-          {data?.page?.properties?.icon || (
-            <FontAwesomeIcon
-              icon={faFile}
-              sx={(theme) => ({
-                fontSize: `${sizes.font}px !important`,
-                color: theme.palette.gray[40],
-              })}
-            />
-          )}
-        </IconButton>
-      </Tooltip>
-      <EmojiPicker
-        popupState={popupState}
-        onEmojiSelect={(emoji) => {
-          void updateEntity({
-            data: {
-              entityId: rewriteEntityIdentifier({
-                accountId,
-                entityId,
-              }),
-              properties: { icon: emoji.native },
-            },
-          });
-        }}
-      />
-    </>
+    <Box
+      sx={{
+        width: sizes.container,
+        height: sizes.container,
+        fontSize: sizes.font,
+        display: "flex",
+        justifyContent: "center",
+        alignItems: "center",
+      }}
+    >
+      {data?.page?.properties?.icon || (
+        <FontAwesomeIcon
+          icon={faFile}
+          sx={(theme) => ({
+            fontSize: `${sizes.font}px !important`,
+            color: theme.palette.gray[40],
```

Comment:
Hovering over the icon button makes the icon hidden since the background color at that point is the same as the font color. Colors (and fontSizes) should be set at IconButton level, since the icon inherits font sizes/colors from it. 

A quick fix will be setting this in PageIconButton, but an ideal fix will be updating our IconButton [styles override](https://github.com/hashintel/hash/blob/main/packages/hash/design-system/src/theme/components/data-display/mui-icon-button-theme-options.ts) to cater for a new "xl" size which is what's needed for the page header icon button. That way we rely on our design system and don't need to manually set font-sizes

I understand this wasn't introduced by this PR, but please can you help fix?

<img width="800" alt="Screenshot 2022-08-24 at 10 53 46" src="https://user-images.githubusercontent.com/16928688/186389873-d2504203-4a92-483a-8d48-e06284101aad.png">